features/00-lab-results.feature

```
@@ -16,31 +16,36 @@ Feature: Lab results
 
     Scenario: Resources have ids
         Given I have a Lab results response
+        And there is at least 1 entry
         Then all resources will have a id field
 
     Scenario: All references will resolve
         Given I have a Lab results response
+        And there is at least 1 entry
         Then all references will resolve
 
     Scenario: There are multiple pages of results
         Given I have a Lab results response
+        And there is at least 1 entry
         When I follow the "next" link
         Then the response code should be 200
 
     Scenario: All the codes are valid
         Given I have a Lab results response
+        And there is at least 1 entry
         Then all the codes will be valid
 
     @warning
     Scenario: Resources fulfill the Argonaut Laboratory Results profile
         Given I have a Lab results response
+        And there is at least 1 entry
         Then there exists one status in Observation.status
         Then there exists one category in Observation.category
         And there exists a fixed Observation.category.coding.system=http://hl7.org/fhir/observation-category
         And there exists a fixed Observation.category.coding.code=laboratory
         Then there exists one code in Observation.code
         And there exists a fixed Observation.code.coding.system=http://loinc.org
-        And Observation.code.coding.code is bound to http://loinc.org/
```

No more silently passing. Yay!